doc/src/dev_intro.rst

```diff
@@ -0,0 +1,152 @@
+MIGraphX Fundamentals
+======================
+
+MIGraphX provides an optimized execution engine for deep learning neural networks.
+We will cover some simple operations in the MIGraphX framework here.
+We created this documentation for contributors to the MIGraphX codebase.
+This documentation is not intended for users of the MIGraphX library.
+
+
+Location of the Examples
+-------------------------
+
+The ``ref_dev_examples.cpp`` can be found in the test directory (``/test``).
+The executable file ``test_ref_dev_examples`` based on this file will be created in the ``bin/`` of the build directory after running ``make -j$(nproc) test_ref_dev_examples``.
+The executable will also be created when running ``make -j$(nproc) check``, alongside with all the other tests.
```

Comment:
Why don't we add a sequence of commands such as:
```
cd build
```
```
make -j$(nproc) test_ref_dev_examples
./bin/test_ref_dev_examples
```
OR
```
make -j$(nproc) check
```

Reply:
There might be a clarity issue where they don't have a build directory or it's named something else. May instead want to link back to the Docker or rbuild instructions.

Reply:
You can link back to the build instructions if needed, but in L13-15 you already mention a build directory. I would just add the commands and say something like "assuming a build directory is already created:..."